include/tvm/relay/attrs/nn.h

```diff
@@ -387,13 +387,16 @@ struct FIFOBufferAttrs : public tvm::AttrsNode<FIFOBufferAttrs> {
 
 /*! \brief Attributes for upsampling operator */
 struct UpSamplingAttrs : public tvm::AttrsNode<UpSamplingAttrs> {
-  int scale;
+  double scale_h;
+  double scale_w;
   std::string layout;
   std::string method;
   bool align_corners;
 
   TVM_DECLARE_ATTRS(UpSamplingAttrs, "relay.attrs.UpSamplingAttrs") {
-    TVM_ATTR_FIELD(scale)
+    TVM_ATTR_FIELD(scale_h)
+        .describe("Should be true to preserve the values at the corner pixels");
```

Comment:
The comment is wrong, it's for `align_corners`. Can you put correct description here?

Reply:
Thanks! I just addressed the review comment.